.changeset/hip-chairs-double.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-tech-radar': minor
+---
+
+Deprecate `RadarEntry.url` - use `RadarEntry.links` instead
```

Comment:
Maybe an example of what the migration should look like would be good here, a before and after :)

Reply:
Good idea, added migration diff. Also updated `sample.ts` to use the `links` attribute instead of `url`.